src/components/Question.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { useState } from "react";
+import { QuestionAnswer, QuestionText, Answer } from "../styles/Apply.styles";
+import Arrow from "../images/arrow.png";
+
+export default function Question({ question }) {
+  const [showQuestion, setShowQuestion] = useState(false);
+
+  return (
+    <QuestionAnswer onClick={() => setShowQuestion(!showQuestion)}>
+      <QuestionText>{question.question} <img src={Arrow} /> </QuestionText>
```

Comment:
add alt text on this image! + split this up so the children are on different lines so code is more readable